addons/account/demo/account_invoice_demo.yml

```diff
@@ -11,11 +11,11 @@
     date_invoice: !eval time.strftime('%Y-%m')+'-01'
     invoice_line_ids:
       - name: 'Dell Inspiron Laptop without OS'
```

Comment:
Maybe just check that name is not automatically computed through the onchange, and removing it would improve the consistency.